manifests/server/conf.pp

```diff
@@ -174,7 +174,7 @@
   Optional[Array[String]]          $macs                            = undef,
   Optional[Boolean]                $passwordauthentication          = undef,
   Boolean                          $permitemptypasswords            = false,
-  Boolean                          $permitrootlogin                 = false,
+  Variant[Boolean,Enum['prohibit-password','without-password','forced-commands-only']] $permitrootlogin = false,
```

Comment:
@bryan-howard First, thanks for the contribution! Unfortunately EL6 does not support the `prohibit-password` option so we will need some logic below to fail on that case.

If you could update this PR, it would be appreciated but, if not, let me know and we'll try to schedule in an out of band update.